.pre-commit-config.yaml

```diff
@@ -0,0 +1,62 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0  # Use the ref you want to point at
```

Comment:
These comments aren't really useful in the config.
```suggestion
    rev: v4.4.0
```

Reply:
Resolved.